README.md

```diff
@@ -153,6 +153,56 @@ acceptance_models = acceptance_model_creator.create_acceptance_map_per_observati
                                                                                    zenith_interpolation=True)
 ```
 
+## Using off runs for background model
+
+It's also possible to create a model from OFF runs and to apply in on runs you want to analyse. The exclusions regions should cover potential sources both in the on and off runs. The OFF runs doesn't need to be connected.
+```python
+acceptance_model_creator = RadialAcceptanceMapCreator(energy_axis_acceptance,
+                                                      offset_axis_acceptance,
+                                                      exclude_regions=exclude_regions,
+                                                      oversample_map=10,
+                                                      min_run_per_cos_zenith_bin=3,
+                                                      initial_cos_zenith_binning=0.01)
+acceptance_models = acceptance_model_creator.create_acceptance_map_per_observation(obs_collection,
+                                                                                   off_observations=obs_collection_off,
```

Comment:
I originally started with this functionality and then extended to the storage part. As it's really minimal in code and could potentially still be usefull, I would keep it.

Reply:
It is easy in code, but I am afraid it could lead to confusion as it is ignored if you provide a base model and it adds one more input parameter. Still, I have no strong opinion on this.

Reply:
I will add a warning for this case to avoid user confusion.